src/main/java/com/hyperwallet/clientsdk/util/HyperwalletApiClient.java

```diff
@@ -182,8 +167,8 @@ private void checkResponseHeader(Response response) {
     }
 
     private String getAuthorizationHeader() {
-        final String pair = this.username + ":" + this.password;
```

Comment:
Why?

Reply:
Can we revert this change. I don't see why it was removed

Reply:
reverted